<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyogramWidget</class>
 <widget class="QWidget" name="MyogramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frSignals">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="Oscilloscope" name="wgtMyoOscill" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblScale">
        <property name="text">
         <string>Масштаб</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbScale"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnMyoRecord">
        <property name="toolTip">
         <string>Сигнал будет записан</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../kernel.qrc">
          <normaloff>:/images/SaveOK.png</normaloff>:/images/SaveOK.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbSubChannels">
        <property name="title">
         <string>Отведения</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Oscilloscope</class>
   <extends>QWidget</extends>
   <header location="global">oscilloscope.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnMyoRecord</sender>
   <signal>clicked(bool)</signal>
   <receiver>MyogramWidget</receiver>
   <slot>on_recMyoClick(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MyogramWidget</receiver>
   <slot>on_myoScaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_recMyoClick(bool)</slot>
  <slot>on_myoScaleChange(int)</slot>
 </slots>
</ui>
